### PR TITLE
Add multipage

### PR DESCRIPTION
**Pull request by jonathanglassman**
### Context

The tech doc tool can handle both single and multipage documentation. 

### Changes proposed in this pull request

Create multipage documentation section

### CODE REVIEW

File: source/architecture/multipage.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+A basic multipage documentation repo can have this structure:
+
+[IMAGE]
+
+You must amend the documentation repo folder structure to reflect this structure.
+
+### Amend the tech-docs.yml file
+
+Add this to the end of your project’s `.../config/tech-docs.yml` file, or set it to true if it is already there:
```
Comment (bahmady) [inline source/architecture/multipage.md]: We shouldn't assume where the user is in relation to their PATH so we can just say about "amending the `tech-docs.yml` file".

File: source/architecture/multipage.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+A nested multipage documentation repo can have this structure:
+
+[IMAGE]
+
+You must amend the documentation repo folder structure to reflect this structure.
+
+### Amend the tech-docs.yml file
+
+Add this to the end of your project’s `.../config/tech-docs.yml` file, or set it to true if it is already there:
```
Comment (bahmady) [inline source/architecture/multipage.md]: Another instance where we should not assume we know where the user is in relation to their PATH

File: source/architecture/multipage.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+
+```
+# Enable multipage navigation in the sidebar
+multipage_nav: true
+```
+### Create multiple .html.md.erb files
```
Comment (bahmady) [inline source/architecture/multipage.md]: Is this information not already included above?

Comment (jonathanglassman) [reply source/architecture/multipage.md]: Duplicating it so each section is self-contained.

File: source/architecture/multipage.md (hunk 1 above)
Comment (jonathanglassman) [reply source/architecture/multipage.md]: agree

File: source/architecture/multipage.md (hunk 2 above)
Comment (jonathanglassman) [reply source/architecture/multipage.md]: agree

**Pull request merged**